static/src/stylesheets/module/football/_overview.scss

```diff
@@ -394,9 +394,11 @@
 .football-knockout-chart__timezone {
     @include fs-textSans(1);
     padding: $gs-baseline 0;
-    float: left;
     text-align: center;
     width: 100%;
+    position: absolute;
+    bottom: 0;
+    z-index: 1000;
```

Comment:
Minor: could we use one of the [z-index vars](https://github.com/guardian/frontend/blob/2de0351556c754998e47fe81b04f9c607ad8006c/static/src/stylesheets/_vars.scss#L288) here? `$zindex-ui` ?